onboarding/github.md

```diff
@@ -0,0 +1,23 @@
+# GitHub
+
+#### Organisation
+
+NeuroPoly has a github organisation:
+https://github.com/neuropoly
+
+#### Purpose
+
+
+The purpose of this organization is to host repos that are related to NeuroPoly's projects. Eg: if you publish a paper, you could put the code+data there and link to it in your paper.
+
+#### Philosophy
+
+The philosophy of NeuroPoly's lab is to do open science, which means that repos should be public. There are some exception: confidential data/credentials, highly-top-secret scientific research, etc.
+
+#### Usage
+
+If you decide to make your repos private, please make sure to not go above the total size and [BW limits](https://docs.github.com/en/billing/managing-billing-for-git-large-file-storage/about-billing-for-git-large-file-storage#purchasing-additional-storage-and-bandwidth)
+
+```{note}
+For students, please always ask your supervisor before creating a repos.
+```
```

Comment:
Maybe this should specify "a private repo"? If the main concern is cost, and cost only applies to private repos, then I don't think we should scare people off from making use of resources. Making a repo shouldn't be a scary thing, it's not a big deal, and in the worst case, we can always just delete it if something really goes bad.